src/main/java/com/conveyal/gtfs/loader/ConditionalRequirement.java

```diff
@@ -0,0 +1,51 @@
+package com.conveyal.gtfs.loader;
+
+/**
+ * These are the values that are checked inline with {@link ConditionalCheckType} to determine if the required
+ * conditions have been met.
+ */
+public class ConditionalRequirement {
+    /** The type of check to be performed on the reference field. */
```

Comment:
Can you elaborate on what the "reference field" is? It isn't immediately clear.

Reply:
Updated.